### ISSUE TITLE
Create new tool to merge MNPs

### ISSUE THREAD

**Issue by vdauwera** [broadinstitute/gatk#1385]
_From @vdauwera on March 30, 2015 13:40_

Create a new walker that reads in phasing information from HC and produces merged MNPs where appropriate. 

_Copied from original issue: broadinstitute/gsa-unstable#908_

**Comment by vdauwera** [comment]
Note that ReadBaskedPhasing doesn't handle indels, only SNPs. If possible it would be good for the new tool to handle indels -- but this is a nice-to-have, not a must-have.

**Comment by vdauwera** [comment]
This would allow us to deprecate ReadBackedPhasing entirely.

**Comment by davidbenjamin** [comment]
@ldgauthier @cdauwera Is there anything left to do before we can say that #4650 closed this?

**Comment by acesnik** [comment]
Is there any plan to have CombineGVCFs handle MNPs? It seems like one could back out the per-base phasing from MNP annotations and then combine the GVCFs.
